test/assignment.coffee

```diff
@@ -141,9 +141,6 @@ test "#1192: assignment starting with object literals", ->
 
 # Destructuring Assignment
 
-test "empty destructuring assignment", ->
-  {} = [] = undefined
-
```

Comment:
Hrm, I did remove the test because destructuring `undefined` or `null` is a runtime error. I think we should be able to restore it with an empty array/object and it should still work... I think.